### DISCUSSION TITLE
workflow/todo/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/workflow#9] [General]
# workflow/todo/

https://shane0.github.io/workflow/todo/

**Comment by shane0** [comment]
# todo

- vim https://shanenull.com/workflow/html/vimfullpage.html
- https://github.com/shane0/workflow/issues

**Reply by shane0** [reply]
do I need issues?  todo cli covers my needs & github lacks vim mode

reminds me I need to find my codemirror vim plugin repo and see if i can implement it here

**Reply by shane0** [reply]
https://github.com/shane0/buddhism/discussions/77

**Reply by shane0** [reply]
# macos 

- touchid missing from settings
- open manually

```sh
x-apple.systempreferences:com.apple.Touch-ID-Settings.extension*TouchIDPasswordPrefs
```

**Comment by shane0** [comment]
# versioned documentation

this needs a jargon decoder & screencast with clear docs and an example repo

https://squidfunk.github.io/mkdocs-material/setup/setting-up-versioning/

demo https://shanenull.com/buddhism/2023/

**Reply by shane0** [reply]
I don’t like the latest alias it breaks comments + I like to start new years fresh so pages aren’t versioned sites are versions

**Comment by shane0** [comment]
create a release

**Reply by shane0** [reply]
releases work but I have no real demand for that feature personally

**Reply by shane0** [reply]
# first principles

- technology is sort of a sprawling mess that is like an onion that never stops growing layers lol
- for example why do people keep adding computer languages when we only need 1?
- https://rosettacode.org/wiki/Rosetta_Code
- maybe ai can solve that but it would need to be open source first
- make a page with a table of problems & solutions or workarounds
- old notes: https://shanenull.com/2023/collapse/collapse/#collapse-of-complex-systems

**Comment by shane0** [comment]
book specific cookiecutter

workflow for notes to flash cards or slideshow

**Comment by shane0** [comment]
add citizen

**Comment by shane0** [comment]
# test drive workflow/

- [ ] screencast: watch it and revise after
- [ ] output: mermaid chart the workflow + upload revised screencast

## workflow

- workflow/ (deploy/update) the cookiecutter into projects
- projects start with a workflow.md
- projects get tasks that can lead to projects.md
- projects.md can lead to cookiecutters
- cookiecutters can have cookiecutters + todo cli + workflow.md + projects.md

**Comment by shane0** [comment]
https://frame.work/

**Reply by shane0** [reply]
to play no mans sky w/ son i needed to update my laptop
i was going to get a frame work but read it has issues running no mans sky
went with mac air m3 15" setting up dev environment going ok so far updated notes

**Comment by shane0** [comment]
# Seven deadly sins

- joe armstrong https://www.youtube.com/watch?v=lKXe3HUG2l4&t=672s

1. Code even you cannot understand a week after
you wrote it - no comments
2. Code with no specifications
3. Code that is shipped as soon as it runs and
before it is beautiful
4. Code with added features
5. Code that is very very fast very very very obscure
and incorrect
6. Code that is not beautiful
7. Code that you wrote without understanding the
problem

**Comment by shane0** [comment]
"Sonoluminescence" is the fancy word for the phenomenon where sound waves create light. It might sound like something out of science fiction, but it's a real scientific phenomenon that can be observed in a lab setting. 
[Image of Sonoluminescence]

Here's how it works:

1. **Sound waves create a bubble in liquid:** A high-frequency sound wave is focused on a small point in a liquid, such as water. This sound wave creates a tiny cavity, or bubble, filled with gas.

2. **The bubble is trapped by the sound waves:** The sound waves keep the bubble from rising to the surface and collapsing. Instead, the sound waves push the bubble in and out, causing it to vibrate.

3. **The bubble collapses rapidly:** As the sound wave intensity increases, the bubble is squeezed to an incredibly small size. This rapid collapse creates a very high temperature and pressure inside the bubble.

4. **Light is emitted:** The extreme conditions inside the collapsing bubble cause it to emit a tiny flash of light. This light is usually blue or ultraviolet, but it can be other colors depending on the gas in the bubble and the conditions of the experiment.

Sonoluminescence is still not fully understood, but scientists believe it may be due to a combination of factors, such as:

* **Thermal radiation:** The high temperature inside the bubble can cause the gas molecules to emit light.
* **Bremsstrahlung radiation:** As the charged particles in the bubble collide, they can emit light.
* **Sonoluminescence is a fascinating phenomenon that demonstrates how sound and light can be interconnected.** It is also being studied for potential applications in areas such as medical imaging and sonochemistry.

**Comment by shane0** [comment]
https://github.com/capjamesg/aurora

**Comment by shane0** [comment]
# experiment: can I make epubs or audiobooks quickly and are they even productive or do they just add time?

- [x] currently researching how to write & publish ebooks & audiobooks will add a links here when it's live

**Reply by shane0** [reply]
- [ ] make a page here & a screencast of the workflow

**Reply by shane0** [reply]
https://play.google.com/books/publish

update: google play is user hostile + many people had the same experience that was a waste of time

**Reply by shane0** [reply]
reflecting on this found publishing epubs on google is user hostile + many users have experienced this i should have researched reviews before bothing that was a big waste of time

might look into other epub / paper book options but it is very low priority

possible productive channels: later when ai makes it easy to do this yourself without dealing with corporate paywalls lock in outsourced moderation user hostile interfaces and so forth

- what I envision: i write in markdown I could collect files dump to an epub or audiobook quickly, publishing on all channels

this does not exist yet I can wait

**Reply by shane0** [reply]
kindle deleted 1984

hosting any content on web or mobile apps is toxic

they charge a %30 vig

you do not actually own the content

they can delete your content or account and there is no recourse

redbox declared bankruptcy and customers lost their purchases

I did create epubs with a table of contents which is the main epub feature i wanted i might do that but i will just stick with markdown and maybe use epubs personally when I’m hiking or offline otherwise

I don’t recommend hosting epubs on corporate app stores

**Comment by shane0** [comment]
# heart sutra cli

- workflow needs requirements.txt

**Comment by shane0** [comment]
# new experiments

- first principles add button to index
- potato chip effect template + tracker
- troubleshooting template + tracker

**Reply by shane0** [reply]
https://htmx.org/docs/
